src/main/java/nl/tudelft/scrumbledore/level/BasicPlayer.java

```diff
@@ -0,0 +1,252 @@
+package nl.tudelft.scrumbledore.level;
+
+import java.util.ArrayList;
+
+import nl.tudelft.scrumbledore.Constants;
+import nl.tudelft.scrumbledore.sprite.Sprite;
+import nl.tudelft.scrumbledore.sprite.SpriteStore;
+
+
+/**
+ * An abstract class as a basic class for representing players.
+ * 
+ * @author Niels Warnars
+ * @author Jesse Tilro
+ * @author David Alderliesten
+ * @author Floris Doolaard
+ */
+public abstract class BasicPlayer extends BasicDynamicElement implements PlayerElement {
```

Comment:
I would call the interface "Player", and the concrete Player "PlayerElement". Also, a BasicPlayer is not needed, since the Player can already be a realization of the Player interface.
<organisation